pkgs/applications/networking/cluster/rke2/README.md

```diff
@@ -1,15 +1,19 @@
 # RKE2 Version
 
-RKE2, Kubernetes, and other clustered software has the property of not being able to update atomically. Most software in nixpkgs, like for example bash, can be updated as part of a `nixos-rebuild switch` without having to worry about the old and the new bash interacting in some way.
+RKE2, Kubernetes, and other clustered software has the property of not being able to update atomically.
+Most software in nixpkgs, like for example bash, can be updated as part of a `nixos-rebuild switch`
+without having to worry about the old and the new bash interacting in some way.
 
 > [!NOTE]
 > Upgrade the server nodes first, one at a time. Once all servers have been upgraded, you may then upgrade agent nodes.
 
 ## Release Channels
 
-RKE2 has there own release channels, which are: `stable`, `latest` and `testing`.
+RKE2 has three main release channels, which are: `stable`, `latest` and `testing`.
```

Comment:
```suggestion
RKE2 has three release channels, which are: `stable`, `latest` and `testing`.
```

Reply:
That's actually correct, there are a lot more release channels, the relevant ("main") ones are these three.